icu4j/main/classes/core/src/com/ibm/icu/number/NumberSkeletonImpl.java

```diff
@@ -1461,6 +1484,15 @@ private static boolean perUnit(MacroProps macros, StringBuilder sb) {
             }
         }
 
+        private static boolean usage(MacroProps macros, StringBuilder sb) {
+            if (macros.usage != null  && macros.usage.length() > 0) {
+                sb.append("usage/" + macros.usage);
```

Comment:
```
sb.append("usage/");
sb.append(macros.usage);
```

Reply:
what is the difference ?

Reply:
`sb.append("usage/" + macros.usage)` expands to approximately

```
StringBuilder sb2 = new StringBuilder();
sb2.append("usage/");
sb2.append(macros.usage);
sb.append(sb2.toString());
```

You should avoid the second StringBuilder.

Reply:
done